modules/LastUserLogin/Feature.php

```diff
@@ -76,11 +76,11 @@ public static function manage_users_custom_column( $value, $column, $user_id ) {
 
 		$last_login = get_user_meta( $user_id, self::META_KEY, true );
 
-		if ( $last_login ) {
-			$value = wp_date( __( 'Y/m/d g:i:s a', 'multisyde' ), $last_login );
+		if ( is_string( $last_login ) && '' !== $last_login ) {
+			return wp_date( __( 'Y/m/d g:i:s a', 'multisyde' ), (int) $last_login );
```

Comment:
I'd just cast this to string and remove the `|false` in return type
```suggestion
			return (string) wp_date( __( 'Y/m/d g:i:s a', 'multisyde' ), (int) $last_login );
```